Apps/phvisualping/visualping.json

```diff
@@ -0,0 +1,536 @@
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.jobid",
+                    "data_type": "string",
+                    "example_values": [
+                        "3003726"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.created",
+                    "data_type": "string",
+                    "example_values": [
+                        "2020-12-21 15:27:48"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.favicon",
+                    "data_type": "string",
+                    "example_values": [
+                        "metadata/_vol_9/_data_924/924307/401942541602d03c6b7e3794d9cbb2f0.png"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.trigger",
+                    "data_type": "string",
+                    "example_values": [
+                        "1"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.interval",
+                    "data_type": "string",
+                    "example_values": [
+                        "1440"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.last_run",
+                    "data_type": "string",
+                    "example_values": [
+                        "2020-12-23 16:30:19"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.active_jobs.*.description",
+                    "data_type": "string",
+                    "example_values": [
+                        "Entry-Specific-Rule"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "show result",
+            "description": "Show the last result of a specific job",
+            "verbose": "Show the details of the last run of a specific visualping job, including a percent difference.",
+            "type": "investigate",
+            "identifier": "show_result",
+            "read_only": true,
+            "parameters": {
+                "id": {
```

Comment:
For Job IDs, you can add a `contains` so that there are contextual actions available

Reply:
Fixed